dapp-oeth/src/components/TransactionListener.js

```diff
@@ -272,7 +271,9 @@ const TransactionListener = ({ rpcProvider }) => {
         // TODO handle a 404 here. We need to retry. A refresh is required to
         // get this to retry at the moment.
         try {
-          safeData = await connector.sdk.txs.getBySafeTxHash(t.hash)
+          // TODO
+          console.log(connector)
```

Comment:
this console.log should be removed, right? it is not needed for users, right?

Reply:
Same comment as the other. Need to find the equivalent of this call from web3-react -> wagmi. The library switch had a gap between them